daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SpeedyTestLib.scala

```diff
@@ -136,5 +137,40 @@ private[speedy] object SpeedyTestLib {
       parserParameter: ParserParameters[X]
   ): PureCompiledPackages =
     typeAndCompile(Map(parserParameter.defaultPackageId -> pkg))
+}
+
+class TestTraceLog extends TraceLog {
```

Comment:
It looks like you no longer need this change?

Reply:
Thanks for that. I'll revert this change.